_config.yml

```diff
@@ -68,8 +68,8 @@ authors:
 
 # Social icons displayed in footer
```

Comment:
For the author, let’s remove the names except Chris, Charles and Vinh. Also please make sure these names are not referred in the docs. Simply replace them to Chris if there’s any.

Reply:
Fixed